src/styles/global.css

```
@@ -1445,6 +1445,104 @@ body.blog-page {
+  margin-bottom: 12px;
+}
+
+.og-heading {
+  font-family: "Cormorant Garamond", Garamond, serif;
+  font-size: 52px;
+  font-weight: 600;
+  line-height: 1.08;
+  color: var(--ink);
+  margin: 0 0 10px;
+  overflow-wrap: break-word;
+  word-break: break-word;
```

Stylelint (17.6.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)